api/src/test/java/org/openmrs/ConceptTest.java

```diff
@@ -23,86 +23,99 @@
 import java.util.Set;
 
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 import org.openmrs.api.APIException;
 import org.openmrs.api.ConceptNameType;
+import org.openmrs.api.ConceptService;
 import org.openmrs.api.context.Context;
 import org.openmrs.test.BaseContextSensitiveTest;
 
+import javax.validation.constraints.AssertTrue;
+
 /**
  * Behavior-driven tests of the Concept class.
  */
 public class ConceptTest extends BaseContextSensitiveTest {
-	
+	protected static final String CONCEPT_XML_DATASET_PACKAGE_PATH = "org/openmrs/api/include/ConceptTest.xml";
+
+	private ConceptService service;
+
+	@Before
+	public void before() throws Exception {
+		service = Context.getConceptService();
+		executeDataSet(CONCEPT_XML_DATASET_PACKAGE_PATH);
+	}
+
 	/**
 	 * When asked for a collection of compatible names, the returned collection should not include
 	 * any incompatible names.
-	 * 
```

Comment:
Can you please revert these auto format changes? Did you format your code?

Reply:
There are no changes when compare with the master branch. 
https://github.com/openmrs/openmrs-core/blob/master/api/src/test/java/org/openmrs/ConceptTest.java
line 37-42

Reply:
It does appear that there are spaces there in master
https://github.com/openmrs/openmrs-core/blob/master/api/src/test/java/org/openmrs/ConceptTest.java#L40

Reply:
Done